src/python_routing_v02/compute_nhd_routing_SingleSeg.py

```diff
@@ -18,10 +18,12 @@
 import numpy as np
 import argparse
 import sys
-sys.path.append(r"../APD/inland_hydraulics/wrf-hydro-run/NWM_2.1_Sample_Datasets/LAKEPARM.nc")
 sys.path.append(r"../../src/fortran_routing/mc_pylink_v00/Reservoir_singleTS")
 import reservoirs_nwm
 from reservoirs_nwm import reservoirs_calc
+import xarray as xr
+import tqdm
```

Comment:
Probably not needed.

Reply:
import tqdm has been removed but the xarray is still used someplace